test/distributed/org/apache/cassandra/distributed/test/cql3/SingleNodeTableWalkTest.java

```diff
@@ -371,10 +372,13 @@ public void test() throws IOException
                                                                         .add(this::selectTokenRange))
                                   .addIf(State::hasEnoughMemtable, StatefulASTBase::flushTable)
                                   .addIf(State::hasEnoughSSTables, StatefulASTBase::compactTable)
+                                  .addAllIf(BaseState::allowRepair, b -> b.add(StatefulASTBase::incrementalRepair)
+                                                                          .add(StatefulASTBase::previewRepair))
```

Comment:
This comment is unrelated to the patch. It might look cleaner if allow composing multiple commands into one, and use `addIf` only. `addAllIf` is not descriptive.

Reply:
sent out PR for this https://github.com/apache/cassandra-accord/pull/189